docs/UserGuide.md

```diff
@@ -360,371 +361,371 @@ edit /c 2 /n cs2040s
 
-Removes a student in your chosen class.
+Marks a student absent for your current lesson.
 
-**Command Format:** `remove /s STUDENT_INDEX /c CLASS_NAME`
+**Command Format:** `unmark /s STUDENT_INDEX /c CLASS_NAME`
 
 **Examples:**
-
-- `remove /s 1 /c CS2103T`: Removes the first student in the class named "CS2103T".
-- `remove /s 2 /c CS2100`: Removes the second student in the class named "CS2100".
+* `unmark /s 2 /c CS2103T` : Marks the second student in your class "CS2103T" as absent.
+* `unmark /s 3 /c CS2102` : Marks the third student in your class "CS2102" as absent.
 
 **Sample Usage:**
 
-1. You want to remove a student named "John" and he is the second student from the class named "CS2103T".
-2. Enter the following command:
-   ```
-   remove /s 2 /c CS2103T
-   ```
+1. Assuming you want to mark the second student named "Jamie" in your class "CS2103T", whereby the class is positioned at index "1" in your class list, as absent.
+2. Enter the following commands:
+```
+view /c 1
+mark /s 2 /c CS2103T
+```
 3. The result box will display the following message:
-   ```
-   John has been removed from CS2103T
-   ```
-4. You have successfully removed John from CS2103T. :tada:
+```
+Jamie has been marked absent!
+```
+4. You have successfully marked Jamie as absent in CS2103T. :tada:
 
 <box type="info">
 
 **Information:**
-- You can identify the STUDENT_INDEX of a student by using the [viewing a class](#viewing-a-class--view) command.
+
+* `CLASS_NAME` must be the name of a class that already exists.
+
+</box>
+
+<box type="warning">
+
+**Caution:**
+
+* Your target student should not already be marked absent.
 
 </box>
 
 <br>
 
-### Editing a student : `edit /s`
+### Marking all students in a class present : `markall /c`
 
-Edits an existing student from your specified class. You can edit the student name, student memo and student participation. This will help keep your student records up-to-date.
+Marks all students in your class as present. You can use this command if all your students are present for your current lesson.
 
-**Command Format:** `edit /s STUDENT_INDEX /c CLASS_NAME [/n STUDENT_NAME] [/id ID] [/m MEMO] [/p PARTICIPATION]`
+**Command Format:** `markall /c CLASS_INDEX`
 
 **Examples:**
-* `edit /s 1 /c CS2103T /n Bob`: Edits the name of the first student in the class CS2103T to "Bob".
-* `edit /s 2 /c CS2103T /id A0231234U`: Edits the id of the second student in the class CS2103T to "A0231234U".
+* `markall /c 1` : Marks all the students in your first class in the class list as present.
+* `markall /c 2` : Marks all the students in your second class in the class list as present.
 
 **Sample Usage:**
 
-1. Assuming you want to edit the ID of the second student to "A010193Z", who is from "CS2103T" class, and the class index in the displayed list is "1".
+1. If you want to mark all your students in the class named "CS2103T", whereby the class is positioned at index "1" in your class list, as present.
```

Comment:
Sample usage sounds good! A few suggestions that I have in mind:
1. You could consider standardising to "Assuming" instead of "If"
2. Change to "...mark all **the** students in **your** class..." to standardise with the sample usage of Marking a Student Present/Absent

Reply:
Thanks for the suggestions! They sounds great, I will make the changes